Tests/PointFreeTests/PricingLandingTests.swift

```diff
@@ -19,7 +19,7 @@ class PricingLandingIntegrationTests: LiveDatabaseTestCase {
   }
 
   func testLanding_LoggedIn_InactiveSubscriber() {
-    var user = User.admin
+    var user = User.mock
```

Comment:
some old tests were using admin mocks because these views used to be guarded for admins only. changing that now